modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache

```diff
@@ -189,7 +189,8 @@ export const {{classname}}AxiosParamCreator = function (configuration?: Configur
             localVarUrlObj.query = {...localVarUrlObj.query, ...localVarQueryParameter, ...options.query};
             // fix override query string Detail: https://stackoverflow.com/a/7517673/1077943
             delete localVarUrlObj.search;
-            localVarRequestOptions.headers = {...localVarHeaderParameter, ...options.headers};
+            let headersFromBaseOptions = baseOptions && baseOptions.headers ? baseOptions.headers : [];
```

Comment:
I think this should initialize to an empty object, right?

Reply:
Yes